core/startos/src/registry/db.rs

```diff
@@ -18,14 +18,19 @@ use crate::util::serde::{apply_expr, HandlerExtSerde};
 
 pub fn db_api<C: Context>() -> ParentHandler<C> {
     ParentHandler::new()
-        .subcommand("dump", from_fn_async(cli_dump).with_display_serializable())
+        .subcommand(
+            "dump",
+            from_fn_async(cli_dump)
+                .with_display_serializable()
+                .with_about("Return db tables and records"),
```

Comment:
👍 I'm glad that you where consistent with the previous db

Reply:
Considering the update of the "dump" sub-command in core/startos/db/mod.rs I have updated this description to "Filter/query db to display tables and records" for consistency.